docs/developers/architecture/app_model.md

```diff
@@ -536,7 +536,10 @@ The {class}`~app_model.backends.qt.QModelMenu` of menu bar items are saved as
 {class}`~napari._qt.qt_main_window.Window` attributes. These are useful as the
 `QAction`s of the commands in these menus can be found via
 the {class}`~app_model.backends.qt.QModelMenu`'s
-{meth}`~app_model.backends.qt.QModelMenu.findAction` method.
+{meth}`~app_model.backends.qt.QModelMenu.findAction` method. `QAction`s of submenu
+of a {class}`~app_model.backends.qt.QModelMenu` currently cannot be found via
+{meth}`~app_model.backends.qt.QModelMenu.findAction`. The utility function
+`get_submenu_action` can be used instead.
```

Comment:
Maybe it could be worthy to write the full path to the `get_submenu_action`? So something like:

```suggestion
{func}`~napari._qt._qapp_model._tests.utils.get_submenu_action` can be used instead.
```

Reply:
I think the test functions don't get an API entry but I'll try and check the built docs. If it doesn't link I'll just put the full path with no `{func}`

Reply:
Link doesn't work so I have just put the whole path there, which is long but I think thats preferable.